mithril-aggregator/src/multi_signer.rs

```diff
@@ -250,7 +255,9 @@ impl MultiSignerImpl {
         {
             Ok(Some(protocol_parameters)) => Ok(Some(protocol_parameters.into())),
             Ok(None) => Ok(None),
-            Err(e) => Err(e.into()),
+            Err(e) => Err(ProtocolError::StoreError(anyhow!(e).context(format!(
+                "Multi Signer can not retrieve protocol parameters for epoch '{epoch}'"
```

Comment:
`context` method can use directly a format string, no need of `format!`.

Reply:
Fixed this one and another I found :+1: